scenarios/smf-setup.xml

```diff
@@ -4,6 +4,10 @@
   -->
 <scenario name="smf-setup" description="Mock SMF setup">
     <group name="Smf-Setup">
+        <step name="Slice-Meter-Config"
+              exec="curl -X POST -H 'Content-Type:application/json' http://localhost:8080/v1/config/network-slices -d@./topo/sliceMeterConfig.json"/>
```

Comment:
to reduce verbosity consider creating a new command under bin to push the network slice config to pfcp-agent. Also, please make sure to respect existing case conventions (we don't use came case for files in the STC directory. I think we only use it for java class files)
```suggestion
              exec="pfcp-agent-push-slice-config topo/pfcp-slice-config.json"/>
```